app/javascript/controllers/application.js

```diff
@@ -0,0 +1,13 @@
+import { Application } from "@hotwired/stimulus";
+
+const application = Application.start();
+
+// Configure Stimulus development experience
+application.debug = false;
+window.Stimulus = application;
+
+export { application };
+
+$(document).ready(function () {
+  $("#exampleModal").modal("show");
+});
```

Comment:
Don't use jQuery, it is not recommended to use nowadays. Try to use vanilla JavaScript instead.